views/js/qtiCreator/widgets/static/math/states/Active.js

```diff
@@ -234,7 +234,8 @@ define([
                 minWidth: 460,
                 maxWidth: 960,
                 minHeight: 220,
-                maxHeight: 640
+                maxHeight: 640,
+                resizableEdges: { top: false, right: true, bottom: true, left: true }
             };
 
         return windowPopupFactory({}, popupOptions)
```

Comment:
In Chrome it's working well, but in Firefox I have an issue - text below popup gets selected. It's not related to your ticket, and it's not a big problem, it's ok for me to leave it like this.

https://user-images.githubusercontent.com/38751932/179707937-fb8ba6bd-2968-4648-a9c9-8286c1cb78a8.mp4